test/WebJobs.Script.Tests/TestScripts/FSharp/ApiHubFileSender/run.fsx

```diff
@@ -2,7 +2,7 @@
 // This prelude allows scripts to be edited in Visual Studio or another F# editing environment 
 
 #if !COMPILED
-#I "../../../../../bin/Binaries/WebJobs.Script.Host"
```

Comment:
I think you should just remove all of these !COMPILED blocks from all the sample/test functions. You don't have to block on this if you feel strongly, but I think it adds more confusion/bloat than it is worth. Furthermore, even with your update I don't get any intellisense - I think your paths are still wrong :) My vote is to just remove and keep the functions clean.